packages/common/client-utils/src/storageUtils.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IPartialSnapshotWithContents } from "@fluidframework/driver-definitions";
+import { ISnapshotTree } from "@fluidframework/protocol-definitions";
+
+/**
+ * Utility API to check if the type of snapshot contents is `IPartialSnapshotWithContents`.
+ * @alpha
+ * @param obj - obj whose type needs to be identified.
+ */
+export function instanceOfIPartialSnapshotWithContents(
```

Comment:
Usually, we start names for functions like that with a verb like "is"
Should it be "isPartialSnapshot()"? I think partial snapshot implies "with content" (unless we will support all combos)

Reply:
Done